src/Azs.Backup.Admin/docs/Azs.Backup.Admin.md

```diff
@@ -23,9 +26,9 @@ Prune the external backup store.
 ### [Restore-AzsBackup](Restore-AzsBackup.md)
 Restore a backup.
 
-### [Set-AzsBackupConfiguration](Set-AzsBackupConfiguration.md)
```

Comment:
Our new API version newly supports the PATCH operation - Update-AzsBackupConfiguration to update the backup configuration, while previously we only have PUT - Set-AzsBackupConfiguration(which actually performed PATCH logic in our service, but now in the service we implement PUT as a real PUT). 
Update-AzsBackupConfiguration is for PATCH, and users can either update a part of the configuration or set it as a whole with this cmdlet.
Clear-AzsBackupConfiguration is for PUT, and users can clear the configuration with this cmdlet. (We restrict this cmdlet to **clearing** since users can already set the whole with Update-AzsBackupConfiguration)

Reply:
thanks for the explanation